src/test/java/net/sf/jabref/logic/util/strings/StringUtilTest.java

```diff
@@ -313,6 +312,9 @@ public void testExpandAuthorInitialsDoNotAddDot() {
         assertEquals("Moore, O.", StringUtil.expandAuthorInitials("Moore, O."));
         assertEquals("Moore, A. O.", StringUtil.expandAuthorInitials("Moore, A. O."));
         assertEquals("Moore, A.-O.", StringUtil.expandAuthorInitials("Moore, A.-O."));
+        assertEquals("MEmre", StringUtil.expandAuthorInitials("MEmre"));
+        assertEquals("{\\'{E}}douard", StringUtil.expandAuthorInitials("{\\'{E}}douard"));
```

Comment:
Makes sense. It might be possible to run RemoveLatexFormatter  (or whatever it is called) and do the detection on the stripped string but probably not worth it for such a corner case.